src/Core/Collision/PolygonalCollider.cpp

```diff
@@ -83,7 +83,8 @@ namespace obe::Collision
     CollisionData PolygonalCollider::getMaximumDistanceBeforeCollision(
         const Transform::UnitVector& offset) const
     {
-        std::vector<Transform::UnitVector> limitedMaxDistances;
+        std::vector<std::pair<PolygonalCollider*, Transform::UnitVector>> collidersInOffset;
```

Comment:
Not sure what you mean by "collidersInOffset" ?
Shouldn't it be collidersWithOffset ?

Reply:
Maybe "reachableColliders" ? That's the colliders that can be touched individually, without taking the other colliders into account.